app/assets/javascripts/application.js

```diff
@@ -11,17 +11,11 @@
 // about supported directives.
 //
 //= require jquery
+//= require bootstrap-sprockets
 //= require jquery_ujs
-//= require jquery.turbolinks
-//= require turbolinks
-//= require turbolinks-compatibility
-//= require jquery-ui/core
+//= require datatables
+//  require turbolinks
```

Comment:
It was mentioned in the PR comments that this was disabled. Do we still need it here ?